app/post_metadata.go

```diff
@@ -549,6 +549,7 @@ func (a *App) getLinkMetadata(requestURL string, timestamp int64, isNewPost bool
 		} else {
 			request.Header.Add("Accept", "image/*")
 			request.Header.Add("Accept", "text/html;q=0.8")
+			request.Header.Add("Accept-Language", *a.Config().LocalizationSettings.DefaultServerLocale)
```

Comment:
There's also a `DefaultClientLocale`... I'm just wondering the difference.

Reply:
@hmhealey This would cause the open graph data to be persisted permanently in the given language, right? And then it would be a separate feature request if we wanted to some day support multi-language preview?

Reply:
From what I can tell, `DefaultClientLocale` is the default language set for new users added to server - whereas `DefaultServerLocale` is the language which system messages are displayed in. These 2 properties seem to correspond to the settings listed in `System Console -> Site Configuration -> Localization`

Reply:
Yeah, that's correct about how `DefaultClientLocale` and `DefaultServerLocale` are used. I don't remember why we didn't just use a single value for both, but that doesn't change much here. I think the server locale would be the correct one to use here.

I suppose we could have this use the post author's locale or even the reader's locale in the future, but yeah, I agree that that would be a separate feature. Doing this much should solve the incorrect preview locale problem for the vast majority of users.